main.go

```diff
@@ -38,8 +42,13 @@ func healthCheckHandler(w http.ResponseWriter, r *http.Request) {
 func corsMiddleware(next http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		origin := r.Header.Get("Origin")
```

Comment:
Using an environment variable here allows for flexibility in different deployment environments without code changes. Good practice.